### DISCUSSION TITLE
so-allow ElasticSearch REST API error

### DISCUSSION THREAD

**Discussion by ashokdev7** [Security-Onion-Solutions/securityonion#6866] [Unsupported Versions] [answered]
I added my IP address in so-allow ElasticSearch REST API 
but When I do : curl - k https://192.168.19.150:9200/_cat/indices

I get error type : security_exception
reason missing authentication credentials for REST request

**Comment by dougburks** [comment, marked as answer]
You will need to authenticate to Elasticsearch. 

`so-elasticsearch-query` makes this easier:
https://docs.securityonion.net/en/2.3/so-elasticsearch-query.html

**Reply by ashokdev7** [reply]
Hi Doug ,
Got it and solved this already 
Just a new Question . like I am using these commands but i want to use http instead of https in elasticsearch REST api 
I have put ssl seeting to false in filebeat.yml and elasticsearch.yml still I am not able to use http 
Can you help in this regard

**Reply by dougburks** [reply]
Sorry, we do not support downgrading https to http.
